### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by ZenoTan**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,4 +2,7 @@
 
 `pyg_lib` could be built as a standalone C++ library. We adopt [CMake](https://cmake.org/) to build and [GoogleTest](https://github.com/google/googletest) to test our C++ code.
 
-The library can be built with `cmake`. If you want to build tests, `-DBUILD_TEST=ON` (`OFF` by default) should be specified, and you can run it with `ctest`. A possible building and testing example could be found [here](https://github.com/pyg-team/pyg-lib/blob/master/.github/workflows/install.yml).
+
+The library can be built with `cmake`. If you want to build tests, `-DBUILD_TEST=ON` (`OFF` by default) should be specified, and you can run them with `ctest`. A possible building and testing example could be found [here](https://github.com/pyg-team/pyg-lib/blob/master/.github/workflows/testing.yml).
+
+You can also build `pyg_lib` as a Python library using `pip install .` with our our C++ library as a `CMakeExtension`, which will be loaded (`libpyg.so`) when you `import pyg_lib`
```
Comment (rusty1s) [inline CONTRIBUTING.md:8]:
```suggestion
You can also build `pyg_lib` as a Python library using `pip install .` with our C++ library as a `CMakeExtension`, which will be loaded (`libpyg.so`) when you `import pyg_lib`
```

File: pyg_lib/csrc/sampler/cpu/random_walk_kernel.cpp [hunk 2]
```diff
@@ -32,7 +32,7 @@ torch::Tensor random_walk_kernel(const torch::Tensor& rowptr,
         for (auto j = 1; j < walk_length + 1; ++j) {
           scalar_t row_start = rowptr_data[v], row_end = rowptr_data[v + 1];
           if (row_end - row_start > 0)
-            v = col_data[randint<scalar_t>(row_start, row_end)];
+            v = col_data[randint(row_start, row_end)];
```
Comment (rusty1s) [inline pyg_lib/csrc/sampler/cpu/random_walk_kernel.cpp:35]: Interesting that this does work without it.

File: pyg_lib/csrc/sampler/cpu/utils.h [hunk 3]
```diff
@@ -4,6 +4,7 @@
 
 template <typename scalar_t>
 inline scalar_t randint(scalar_t low, scalar_t high) {
+  TORCH_CHECK(low <= high, "Illegal random range");
```
Comment (rusty1s) [inline pyg_lib/csrc/sampler/cpu/utils.h:7]: This should be caught by `libtorch`, so I removed it.

**Review by rusty1s — APPROVED** [review]

File: pyg_lib/csrc/sampler/cpu/random_walk_kernel.cpp (hunk 2 above)
Comment (ZenoTan) [reply pyg_lib/csrc/sampler/cpu/random_walk_kernel.cpp:35]: Function templates can induce the types by examining args, making coding easier :)

**Pull request merged**